Sources/IssueReporting/Internal/SwiftTesting.swift

```diff
@@ -87,7 +67,7 @@ func _recordError(
             $s7Testing5IssueV6record__14sourceLocationACs5Error_p_AA7CommentVSgAA06SourceE0VtFZ
             """,
           in: "Testing",
-          to: (@convention(thin) (any Error, Any?, SourceLocation) -> Any).self
+          to: (@convention(thin) (any Error, Any?, SourceLocation) -> Issue).self
```

Comment:
Returning `Any` here was causing some Swift crashes related to attempting to destroy an existential box that I suppose doesn't exist. Luckily adding an `Issue` shim seems to work just fine.